requirements/requirements.txt

```
@@ -1,19 +1,19 @@
 colorama
-distrax @ git+https://github.com/google-deepmind/distrax  # distrax release doesn't support jax > 0.4.13
+distrax
 flashbax~=0.1.0
 flax
 gigastep @ git+https://github.com/mlech26l/gigastep
 hydra-core==1.3.2
 id-marl-eval @ git+https://github.com/instadeepai/marl-eval
-jax
-jaxlib
+jax==0.4.25  # mabrax limiting the version here :/
+jaxlib==0.4.25
 jaxmarl
-jumanji @ git+https://github.com/sash-a/jumanji
+jumanji @ git+https://github.com/sash-a/jumanji  # there are a few extra MARL envs here
```

Is it a few more or more and modified? 🤔 

```suggestion
jumanji @ git+https://github.com/sash-a/jumanji  # # Includes a few extra MARL envs
```